Assets/Plugins/Source/Editor/ConfigEditors/ConfigEditor.cs

```diff
@@ -261,6 +261,9 @@ protected void RenderConfigFields()
                                 onClick();
                             }
                             break;
+                        case ConfigFieldType.Enum:
+                            field.FieldInfo.SetValue(config, GUIEditorUtility.RenderInputField(field.FieldDetails, field.FieldInfo.GetValue(config) as Enum, labelWidth));
+                            break;
```

Comment:
I think that until we have a need for this `case`, it should not be included. This is in context with other comments made in this review. \*This\* conversation can be marked as resolved so long as the others are discussed.